hbogolib/handlereu.py

```diff
@@ -1294,3 +1310,47 @@ def addCat(self, name, url, icon, mode):
         liz.setInfo(type="Video", infoLabels={"Title": name})
         liz.setProperty('isPlayable', "false")
         xbmcplugin.addDirectoryItem(handle=self.handle, url=category_url, listitem=liz, isFolder=True)
+
+    def get_elapsed(self, ExternalId):
+        Position = 0
+        for list in self.JsonHis:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'list'](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4677960)